### PR TITLE
Add initial support for more buildings

### LINKED ISSUE #14: The `process` subcommand only works for the chiller-based TES
The `process` subcommand only works for the chiller-based TES data and crashes out for the DX coil TES.

### PR DESCRIPTION
**Pull request by jasondegraw**
This PR adds initial support for most of the rest of the prototypes (with the exception of the apartments and the small data center). A new section in the README lists out what is supported by what, which at this point has no overlap. The PR also builds out a bit of a testing setup, but the CLI ThermalTank output isn't quite right to get a zero-diff match with the API output (yet).

Other changes include improvements to the `process` subcommand (which now handles both of the TES types) and changes to the example file organization.

Closes #14

### CODE REVIEW

File: examples/run-api-examples.py
Comment (jasondegraw) [inline examples/run-api-examples.py:1]: I'm moving some of this over to the `regresssion_test` directory. This file won't come back, but others might.

File: regression_tests/run_tests.py
Comment (jasondegraw) [inline regression_tests/run_tests.py:1]: This file is currently a plain script, but I'll change it over to be a pytest-based script at some point.

File: regression_tests/College_3A_2007_thermaltank_defaults_cli.csv
Comment (jasondegraw) [inline regression_tests/College_3A_2007_thermaltank_defaults_cli.csv:1]: All of these results files may not really belong here, but I'll put them here for now.

File: regression_tests/tests.json
Comment (jasondegraw) [inline regression_tests/tests.json:1]: This file contains all of the tests for the CLI and the API in JSON form. It should be pretty self explanatory.

File: src/stor4build/cli/__init__.py [hunk 1]
```diff
@@ -71,32 +77,74 @@ def process(csvfile, date): #osm, epw, openstudio, measures_dir, measures_only,
...
-    assert(len(baseline_cols) == len(tes_cols))
+
     df['date'] = df['Date/Time'].dt.date
-    click.echo(df)
-    dfx = df[df['date'] == date]
-    click.echo(dfx)
-    baseline = dfx[baseline_cols].sum(axis=1)
-    tes = dfx[tes_cols].sum(axis=1)
+
+    if 'storage' in info and info['storage'] == 'PackagedIceStorage':
```
Comment (jasondegraw) [inline src/stor4build/cli/__init__.py:83]: The `process` script can now handle both the TES systems we support.

File: src/stor4build/util.py [hunk 2]
```diff
@@ -173,3 +191,43 @@ def read_results(csv_file:str, echo=print):
     echo(df)
     return header_info, df
 
+def prepare_detailed_header(building_type, climate_zone, vintage, storage_type, storage_medium=None, report_dir=None,
```
Comment (jasondegraw) [inline src/stor4build/util.py:194]: This new function should eventually handle the creation of the header info for both of the TES systems and replace the code that is in a couple of places.

File: README.md [hunk 3]
```diff
@@ -82,6 +82,34 @@ flask --app stor4build.api run
 
 This will start up the flask **development** server and output will appear on the console. The API accepts JSON inputs in the form described in the `schema` directory in the file `stor4build.json`. Example inputs and scripts to send them to the API are in the `resources` and `scripts` directories.
 
+## Prototype Coverage
```
Comment (jasondegraw) [inline README.md:85]: The table lists out the current support of the prototypes.

**Review by jasondegraw — COMMENTED** [review]
Walkthrough of the changes.

**Review by bonnema — APPROVED** [review]
